hotel.pug

```diff
@@ -19,9 +19,9 @@ block content
         h1 Marriott at Research Triangle Park
 
         picture
-            source(srcset='/img/hotel/marriott-lobby.webp' type='image/webp')
-            source(srcset='/img/hotel/marriott-lobby.jpg' type='image/jpeg')
-            img(class='img-fluid mt-3 mb-3' src='/img/hotel/marriott-lobby.webp' alt='Marriott at RTP')
+            source(srcset='/img/hotel/marriott-outside-dusk.webp' type='image/webp')
+            source(srcset='/img/hotel/marriott-outside-dusk.jpg' type='image/jpeg')
+            img(id='hotel-img' class='my-3 mx-auto' src='/img/hotel/marriott-outside-dusk.webp' alt='Marriott at RTP')
```

Comment:
Instead of separate `mt-3` and `mb-3`, we can use the combined `my-3`.

Reply:
Also here, which is more appropriate, including things in the scss for the id tag, or using bootstrap classes?  Thinking about the `display: block` part.